apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -172,9 +235,10 @@ maybe_create_vtep(#{<<"backend">> := Backend}, #state{netlink = Pid}) ->
           ok = try_enable_ipv6(VTEPNameStr),
           {ParsedVTEPIP6, PrefixLen6} = parse_subnet(VTEPIP6),
           {ok, _} = dcos_overlay_netlink:ipaddr_replace(Pid, inet6, ParsedVTEPIP6, PrefixLen6, VTEPNameStr),
-          lager:notice("Overlay VTEP was configured, ~s => ~p", [VTEPName, Info#{ip => VTEPIP6}])
+          lager:notice("Overlay VTEP address was configured, ~s => ~p", [VTEPName, Info#{ipv6 => VTEPIP6}])
```

Comment:
`Info#{ip => VTEPIP6}` was a correct value. We don't use ipv4 address for ipv6 vtep addr, that's why `ip` must be overridden.

Reply:
Also `Info` now contains only one `ip` value, so it would be better to simplify logging in `create_vtep_addr/2`.

```
lager:notice("Overlay VTEP address was configured, ~s => ~p", [VTEPName, VTEPIP]),
…
lager:notice("Overlay VTEP address was configured, ~s => ~p", [VTEPName, VTEPIP6])
```

Reply:
It seems IPv4 is a must and ipv4 is optional.
It first created an ipv4 address and create a coexisting ipv6 address optionally? So I should log i configured a ipv4 address, and then i configured an ipv6 address.

Reply:
yup, that's how it is right now, but your patch is removing [this](https://github.com/dcos/dcos-net/pull/81/files#diff-8c19cd0256d5151b9727cdde5bbb7de8L165) line (see it in split mode). it should be one log message for ipv4 and one log message for ipv6.